templates/index.html

```diff
@@ -1,6 +1,6 @@
-{{! This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at https://mozilla.org/MPL/2.0/. }}
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
```

Comment:
Do you think we should send this to everyone requesting our web page? I'd prefer a template comment.

Reply:
I don't remember why I changed the comments... but I must had trouble with them.

I'm gonna revisit this patch later.

Reply:
I remember why I switched this to a comment. Because I removed the comment from the SVG files.